### DISCUSSION TITLE
Enable field name translations to use "Custom translations"

### DISCUSSION THREAD

**Discussion by ruihildt** [directus/directus#19226] [Feature Requests]
### Summary

Custom translations are useful because you can centralize all translations in the same place.

### Motivation

Unfortunately, field name translations can't use the Custom translations.
So sometimes you will have to duplicate translations, and not every translations will be centralized in the same place.

### Detailed Design

Next to the "Create new" button (or instead), add the translation icon to make it possible to select an existing custom translation

### Must Have:
- easy pick from the Custom translations

### Drawbacks

No drawbacks. :)

### Alternatives

Don't do anything and keep the issue unsolved.

### Adoption Strategy

If you already have translation, I guess it would be nice to offer to add them to the Custom translations.
Ideally this could be automated.

We could imagine to auto-migrate all existing Field name translations, and use as the translation key the field key, and if there's already something with that key, add a simple increment (so field_1, field_2, etc.).

**Comment by rijkvanzanten** [comment]
Yes 100%. I actually thought this was already an open issue!

**Reply by rijkvanzanten** [reply]
I'll move this to accepted and open an issue to get this implemented 🚀

**Reply by ruihildt** [reply]
Great to hear. :)

**Comment by rijkvanzanten** [comment]
This feature request was accepted. Implementation will be tracked here! https://github.com/directus/directus/issues/19228
